splunk_connector.py

```diff
@@ -89,33 +89,33 @@ def _get_error_message_from_exception(self, e):
         :return: error message
         """
         error_code = None
-        error_msg = consts.SPLUNK_ERR_MSG_UNAVAILABLE
+        error_message = consts.SPLUNK_ERR_MESSAGE_UNAVAILABLE
 
         self.error_print("Traceback: {}".format(traceback.format_stack()))
         try:
             if hasattr(e, "args"):
                 if len(e.args) > 1:
                     error_code = e.args[0]
-                    error_msg = e.args[1]
+                    error_message = e.args[1]
                 elif len(e.args) == 1:
-                    error_msg = e.args[0]
+                    error_message = e.args[0]
             else:
-                error_msg = consts.SPLUNK_ERR_MSG_UNAVAILABLE
+                error_message = consts.SPLUNK_ERR_MESSAGE_UNAVAILABLE
```

Comment:
Should use ERROR_MESSAGE or ERR_MSG. There is a new automation check that I believe will catch this wording mismatch - ERR (shortened word) and MESSAGE (full word)

Reply:
Hi @alexa-phantom 
thanks for the feedback, we have verified and the pipelines are passing with new automation checks.

I still agree with your point to have uniformity in terms of variable names, as this build is being anticipated by multiple customers (~4/5 customer bugs will be resolved with this release) and the changes suggested are not functional changes, we would be proceeding with the release and will fix the variables in the upcoming release.